cmd/config-encrypted.go

```diff
@@ -172,9 +171,6 @@ func getOldCreds() (activeCredOld auth.Credentials, err error) {
 		if err != nil {
 			return activeCredOld, err
 		}
-		// Once we have obtained the rotating creds
-		os.Unsetenv(config.EnvAccessKeyOld)
```

Comment:
We should unset this automatically @vadmeste - what is the reason for this removal?

Reply:
> We should unset this automatically @vadmeste - what is the reason for this removal?

getOldCreds() is called from multiple places, shall we store them in the memory and unset ?

Reply:
Yes because we don't want any other code to be looking for this value @vadmeste

Reply:
Most users forget to unset this value, so to avoid any leakage..